sdk/js/src/token_bridge/getIsWrappedAsset.ts

```diff
@@ -116,16 +121,25 @@ export function getIsWrappedAssetNear(
   return asset.endsWith("." + tokenBridge);
 }
 
-// TODO: do we need to check if token is registered in bridge?
+/**
+ * Determines whether or not given address is wrapped or native to Aptos.
+ * @param client Client used to transfer data to/from Aptos node
+ * @param tokenBridgeAddress Address of token bridge
+ * @param assetFullyQualifiedType Fully qualified type of asset
+ * @returns True if asset is wrapped
+ */
 export async function getIsWrappedAssetAptos(
   client: AptosClient,
   tokenBridgeAddress: string,
-  assetAddress: string,
+  assetFullyQualifiedType: string
 ): Promise<boolean> {
-  assetAddress = ensureHexPrefix(assetAddress);
+  assetFullyQualifiedType = ensureHexPrefix(assetFullyQualifiedType);
   try {
     // get origin info from asset address
-    await client.getAccountResource(assetAddress, `${tokenBridgeAddress}::state::OriginInfo`);
+    await client.getAccountResource(
```

Comment:
Same comment. It doesn’t seem unreasonable to know the address from a type in Aptos.

Reply:
Actually, this seems backwards compatible since all it’s doing is a split.